the-wing/components
# Mo 236/carousel

## Description
Add carousel component (responsive)

## Jira Ticket
[MO-236](https://thewing.atlassian.net/browse/MO-236)

## How should this be manually tested?
1. Go to carousel story
2. Visit default
3. Click "Viewport" in addons tab (you'll see "Action logger" and "Viewport")
4. Make sure that carousel looks correct on all screen sizes
5. Repeat for "1 slide" story

## Type of change


## Checklist:

~- [ ] I have written tests or updated existing tests for this change~ *update when testing harness has been added*

~- [ ] New and existing tests pass locally~ *update when testing harness has been added*
